charts/foundations/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.1.21
+version: 0.1.22
```

Comment:
I guess it's a breaking change tho because of the move?

Reply:
The actual chart itself doesn't break, so it can be auto-upgraded safely.

We'd need to take care when upgrading the `foundations` file, but I can't exactly control that except do them myself.